include/exercise_functions.php

```diff
@@ -0,0 +1,504 @@
+    ";
+
+  }
+
+}
+
+
+function getExercises(){
+
+  $userId = $_SESSION['userId'];
+
+  $exerciseData = db_Query("
+    SELECT *
+    FROM exercise
+    WHERE userId = '$userId'
+  ")->fetchAll();
+
+  return $exerciseData;
+
+}
+
+function getSingleExercise($exerciseIdPass){
+
+  $exerciseData = db_Query("
+    SELECT *
+    FROM exercise
+    WHERE exerciseId = '$exerciseIdPass' 
+  ")->fetch();
+
+  return $exerciseData;
+
+}
+
+function getSessions(){
+
+  $sessionData = db_Query("
+    SELECT *
+    FROM session
+  ")->fetchAll();
+
+  return $sessionData;
+
+}
+
+function displayAllExercises(){
+
+  $allExercises = getExercises();
+  
+  foreach($allExercises as $individualExercise){
+    $exerciseTypeName = getExerciseTypeName($individualExercise);
+
+    echo"
+      <label for=".$individualExercise['exerciseId']." />
+        <form action='log_exercise.php' method='post'>
+          <input type='hidden' name='exerciseIdPass' value=".$individualExercise['exerciseId']." />
+          <input type='submit' class='displayed_exercise_submit_button' id=".$individualExercise['exerciseId']." value=''/>
+          <div class='exercise_display_card'>
+            <div class='exercise_display_namebox'>
+              <h3 class='exercise_display_name'>".$individualExercise['exerciseName']."</h3>
+            </div>
+            <div class='exercise_display_typebox'>
+              <h3 class='exercise_display_type'>".$exerciseTypeName."</h3>
+            </div>
+          </div>
+        </form>
+      </label>
+    ";
+  }
+}
+
+function displayLogPage($exerciseData, $nextExerciseId){
+
+  if($exerciseData['typeId'] == 1){
+    displayCardio($nextExerciseId);
+  }
+
+  else if($exerciseData['typeId'] == 2){
+    displayWeights($nextExerciseId);
+  }
+
+  else if($exerciseData['typeId'] == 3){
+    displayBodyweight($nextExerciseId);
+  }
+
+}
+
+function displayCardio(){
+  echo"
+  <div id='current_attempt'>
+    
+      <h3 class='log_attempt_header'>Current Attempt</h3>
+      <div id='log_exercise_form_layout'>
+  
+        <form id='log_exercise_form'>
+          <div id='log_exercise_two_box_form'>
+    
+            <div class='log_page_formrow'>
+              <h3 class='log_exercise_input_labels'>Minutes:</h3>
+              <input type='number' name='minutes' class='log_exercise_formbox' />
+            </div>
+      
+            <div class='log_page_formrow'>
+              <h3 class='log_exercise_input_labels'>Distance:</h3>
+              <input type='number' name='distance' class='log_exercise_formbox' />
+            </div>
+      
+          </div>
+  
+        <input type='submit' id='log_exercise_submit' name='log_exercise'/>
+  
+        </form>
+  
+        <div id='log_page_buttons'>
+  
+          <div id='log_exercise_button'>
+            <label id='log_exercise_button_label' for='log_exercise_submit' tabindex='0'>Log Exercise</label>
```

Comment:
If this is a label for the button in the form why is it not near the button? Also you could just give the actual button text.